### DISCUSSION TITLE
Add the 'Quick add' button to related lists

### DISCUSSION THREAD

**Discussion by CritchCode** [arnoudkooi/SN-Utils#451] [Ideas]
Hi Arnoud, it was great to meet you at CreatorCon and help out in the labs. I was amazed at your memory about my dropbox issue! You must have so many people come up to you and also hit you with issues and discussions. Well played!

<img width="226" alt="image" src="https://github.com/arnoudkooi/ServiceNow-Utils/assets/55980966/f7a3f98e-fd32-4633-946a-db5cf7e2f7d3">

I have an idea for an enhancement. I use the 'Quick add' buttons a LOT, especially since you added the sys_id options. I was wondering if it would be possible to add the quick add button to the related lists?

![image](https://github.com/arnoudkooi/ServiceNow-Utils/assets/55980966/00f993b1-5859-4231-b2a3-e20dacf5ece2)

Thanks :-)

**Comment by arnoudkooi** [comment]
Great meeting you! 
I'll take a look. The problem is related lists often load async, so may be easiest to link it to the /tn function for simplicity.

**Comment by arnoudkooi** [comment]
This should work starting version 7.1.8.0
Please check if it works.
When you load related lists on demand or async, the button should appear when you mouse in the related list area.

**Comment by arnoudkooi** [comment]
btw, I also published 7.1.8.0 to the Safari App Store, coming from 7.1.3.6

**Comment by arnoudkooi** [comment]
Did you get the memo? ;)

**Comment by CritchCode** [comment]
Sorry, I didn't get the notification for some reason. I just tried this out and it works perfectly! Exactly as I'd hoped and works on reference related lists as well as scripted relations. Spot on, perfect! Thanks Arnoud
